### DISCUSSION TITLE
need help for formal verifiaction of picorv32

### DISCUSSION THREAD

**Discussion by Vinayakamk** [Z3Prover/z3#6945] [General]
hi,
this is more into clarification rather an issue.

i wish to understand whats happening in my run for picorv32.
![image](https://github.com/Z3Prover/z3/assets/137029135/1146c4e4-07c5-4bb0-bc79-1e568450de69)

this was the comtent of my sby file.
[options]
mode prove
depth 10

[engines]
smtbmc z3

[script]
read_verilog -formal picorv32a.v
prep -top picorv32

[files] 
picorv32a.v

and picorv32 is used from openlane/designs (yosys)

**Comment by NikolajBjorner** [comment]
This is not the right place to get help on smtbmc or yosys tools. Better to file the question on some place that relates to smtbmc.
